src/util-base64.c

```diff
@@ -93,68 +94,136 @@ static inline void DecodeBase64Block(uint8_t ascii[ASCII_BLOCK], uint8_t b64[B64
     ascii[2] = (uint8_t) (b64[2] << 6) | (b64[3]);
 }
 
+static inline bool AmpleDestinationSize(
```

Comment:
Seems overkill to have such a simple function :-p

Reply:
Indeed. :P Was hoping to do more things at first here. Now, it's not much use. Will remove. :)